groovy-language-server/src/main/java/com/palantir/groovylanguageserver/GroovycWrapper.java

```diff
@@ -116,6 +131,61 @@ public Path getWorkspaceRoot() {
     }
 
     @Override
+    public Map<String, Set<SymbolInformation>> getReferences() {
+        return references;
```

Comment:
Yep I think that's fine! The same will happen with `getFileSymbols`. The alternative would be to have them both throw an exception if `compile()` was never called successfully (without compilation errors) before this was called. Thoughts?